server/test/services/applicant/BlockTest.java

```diff
@@ -468,12 +468,11 @@ public void hasRequiredQuestionsThatAreSkippedInCurrentProgram_returnsTrue() {
         .map(ApplicantQuestion::getContextualizedPath)
         .forEach(path -> QuestionAnswerer.addMetadata(applicantData, path, programId, 1L));
 
-    assertThat(block.hasRequiredQuestionsThatAreSkippedInCurrentProgram()).isTrue();
+    assertThat(block.hasErrors()).isTrue();
```

Comment:
Do we have the ability to assert that the "this question is required" error appears in there or is there not enough visibility for that assertion?

Reply:
Good idea! Done